Mailbox/Program.cs

```diff
@@ -89,17 +89,51 @@ static void Main(string[] args)
 
         public static string GetOwnersDisplay(Mailboxes mailboxes)
         {
-            
+            string listNames = "";
+
+            foreach(Mailbox mailbox in mailboxes)
+            {
+                listNames += mailbox.Owner + ", ";
+            }
+
+            return listNames;
         }
 
         public static string GetMailboxDetails(Mailboxes mailboxes, int x, int y)
         {
             
+            foreach(Mailbox mailbox in mailboxes)
+            {
+                if (mailbox.Location.Equals(new ValueTuple<int,int>(x,y)))
```

Comment:
Simplify tuple syntax
```suggestion
                if (mailbox.Location.Equals((x,y)))
```